wgsl/index.bs

```diff
@@ -632,16 +633,71 @@ module header and each usage of a GLSL method will add the appropriate
   </xmp>
 </div>
 
+## Module Constants ## {#module-constants}
+
+A *module constant* declares a name for a value, outside of all function declarations.
+The name is available for use after the end of the declaration,
+until the end of the WGSL program.
+
+When the declaration has no attributes, the value is the expression in the declaration.
```

Comment:
Should we note that the initializer is required?

Reply:
Well, the grammar requires there to be an initializer expression.  So I don't think we need to say extra here.